extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php

```diff
@@ -111,103 +107,21 @@ public function sendDigestToUser( $userID ) {
 			return;
 		}
 
-		$dbr = wfGetDB( DB_SLAVE, [], \F::app()->wg->ExternalDatawareDB );
-
-		$oResource = $dbr->select(
-			[ GlobalWatchlistTable::TABLE_NAME ],
-			[
-				GlobalWatchlistTable::COLUMN_ID,
-				GlobalWatchlistTable::COLUMN_USER_ID,
-				GlobalWatchlistTable::COLUMN_CITY_ID,
-				GlobalWatchlistTable::COLUMN_NAMESPACE,
-				GlobalWatchlistTable::COLUMN_TITLE,
-				GlobalWatchlistTable::COLUMN_REVISION_ID,
-				GlobalWatchlistTable::COLUMN_TIMESTAMP
-			],
-			[ GlobalWatchlistTable::COLUMN_USER_ID => $userID ],
-			__METHOD__,
-			[ "ORDER BY" => GlobalWatchlistTable::COLUMN_TIMESTAMP . ", " . GlobalWatchlistTable::COLUMN_CITY_ID ]
-		);
-
-		$records = $dbr->numRows( $oResource );
-		$bTooManyPages = ( $records > self::MAX_ARTICLES_PER_WIKI );
-		$iWikiId = $loop = 0;
-		$aDigestData = [];
-		$aWikiDigest = [ 'pages' => [] ];
-		$aRemove = [];
-		while ( $oResultRow = $dbr->fetchObject( $oResource ) ) {
-			# ---
-			if ( $loop >= self::MAX_ARTICLES_PER_WIKI ) {
-				break;
-			}
-
-			$oWikia = WikiFactory::getWikiByID( $oResultRow->gwa_city_id );
-			if ( empty( $oWikia->city_public ) ) {
-				continue;
-			}
-
-			if ( $iWikiId != $oResultRow->gwa_city_id ) {
-
-				if ( count( $aWikiDigest['pages'] ) ) {
-					$aDigestData[ $iWikiId ] = $aWikiDigest;
-				}
-
-				$iWikiId = $oResultRow->gwa_city_id;
-
-				if ( isset( $aDigestData[ $iWikiId ] ) ) {
-					$aWikiDigest = $aDigestData[$iWikiId];
-				} else {
-					$aWikiDigest = [
-						'wikiName' => $oWikia->city_title,
-						'wikiLangCode' => $oWikia->city_lang,
-						'pages' => []
-					];
-				}
-			} // if
-
-			if ( in_array( $oResultRow->gwa_namespace, [ NS_BLOG_ARTICLE_TALK, NS_BLOG_ARTICLE ] ) ) {
```

Comment:
The old weekly digest email gave special treatment to blogs and article comments. It called them out in a separate section of the email. The new designs don't draw any distinction here, all changes (article, category, blogs. etc) are displayed in the same way.

I double checked with Cristina and she said that's the behavior we want, so this logic, and all the logic found in `makeBlogsList()` can be removed.